_includes/v20.1/misc/movr-workflow.md

```diff
@@ -1,23 +1,42 @@
-The workflow for MovR is as follows (with approximations of the corresponding SQL for each step):
+The workflow for MovR is as follows (with approximations of the corresponding SQL and some executable examples for each step):
```

Comment:
I'd remove the parenthetical statement, so that it reads as follows:

`The workflow for MovR is as follows:`

Reply:
Removed.